circle.yml

```diff
@@ -19,11 +19,11 @@ test:
       swiftlint lint --strict --reporter json |
       tee $CIRCLE_ARTIFACTS/swiftlint-report.json
     - bin/test iOS 9.3
-    - bin/test iOS 10.1
+    - bin/test iOS 10.2
     - bin/test UIKit-iOS 9.3
-    - bin/test UIKit-iOS 10.1
-    - bin/test tvOS 10.1
-    - bin/test UIKit-tvOS 10.1
+    - bin/test UIKit-iOS 10.2
+    - bin/test tvOS 10.0
+    - bin/test UIKit-tvOS 10.0
```

Comment:
had to mess with these OS's cause circle keeps changing which simulators they have installed :/